app_vue/.prettierrc.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://json.schemastore.org/prettierrc",
-  "semi": false,
+  "semi": true,
```

Comment:
updating for us to become pro-semi colon at the end of lines in javascript 😄  (can change if we want)